int32 found_line 
int32 center
float64 slope
int32[] closest_white_pixel
 